schema/definitions/0.8.0/schema/fmu_results.json

```diff
@@ -428,6 +429,26 @@
                             "examples": [
                                 1
                             ]
+                        },
+                        "size": {
```

Comment:
Is this different from `file.size_bytes`?

Reply:
Yes, the `file.size_bytes` refers to the materialized file, while `data.size` refers to the content in the file.